db/rocksdb.go

```diff
@@ -30,6 +30,15 @@ const maxAddrDescLen = 1024
 // when doing huge scan, it is better to close it and reopen from time to time to free the resources
 const refreshIterator = 5000000
 
+// CoinGeckoTimeFormat is a format string for storing FiatRates timestamps in rocksdb
+const CoinGeckoTimeFormat = "20060102030405" // YYYYMMDDhhmmss
```

Comment:
Do not call it CoinGeckoTimeFormat, it will be same for other possible providers of fiat rate. Call it FiatRateTimeFormat or something similar

Reply:
Fixed in https://github.com/trezor/blockbook/pull/316/commits/01613892e734abd2d92c4c86a1ac067c62cdfd86